docs/core/tools/dotnet-new-install.md

```diff
@@ -31,6 +31,23 @@ For example, if you install the template package using `dotnet new --install` in
 
 Starting with .NET SDK 6.0.100, installed template packages will available in later .NET SDK versions installed on your machine. The template package installed in .NET SDK 6.0.100 will also be available in .NET SDK 6.0.101, .NET SDK 6.0.200 and so on. However these template packages won't be available in .NET SDK versions prior to .NET SDK 6.0.100. To use template package installed .NET SDK 6.0.100 or later  in earlier .NET SDK versions, you need to install it using `dotnet new --install` using that .NET SDK version.
 
+<!-- markdownlint-disable MD012 -->
+[!INCLUDE [new syntax](../../../includes/dotnet-new-7-0-syntax.md)]
+
+Examples:
+
+- Show help for install subcommand
+
+```
+dotnet new install --help
```

Comment:
everywhere there's a block like this introduced by a bullet point, the block should be indented two spaces so the rendered example text will appear at the same indentation as the bullet text.

Reply:
fixed